pgdb/v1/fts_test.go

```diff
@@ -69,6 +69,43 @@ func TestSearchEmpty(t *testing.T) {
 	requireQueryFalse(t, pg, vector, " ")
 }
 
+func TestSearchCamelCase(t *testing.T) {
+	pg, err := pgtest.Start()
+	require.NoError(t, err)
+	defer pg.Stop()
+
+	vector := FullTextSearchVectors([]*SearchContent{
+		{
+			Type:   FieldOptions_FULL_TEXT_TYPE_ENGLISH,
+			Weight: FieldOptions_FULL_TEXT_WEIGHT_HIGH,
+			Value:  "AWSProdUSEastAuthSNSTestSQS FooBarLOLFoobarSOSTestNOPE CheezeBreeze",
+		},
+	})
+
+	requireQueryTrue(t, pg, vector, "awsproduseastauthsnstestsqs")
+	requireQueryTrue(t, pg, vector, "prod")
+	requireQueryTrue(t, pg, vector, "east")
+	requireQueryTrue(t, pg, vector, "auth")
+	requireQueryTrue(t, pg, vector, "test")
+	requireQueryTrue(t, pg, vector, "foo")
+	requireQueryTrue(t, pg, vector, "bar")
+	requireQueryTrue(t, pg, vector, "foobar")
+	requireQueryTrue(t, pg, vector, "test")
+	requireQueryTrue(t, pg, vector, "cheeze")
+	requireQueryTrue(t, pg, vector, "breeze")
+	requireQueryTrue(t, pg, vector, "aws")
+	requireQueryTrue(t, pg, vector, "us")
+	requireQueryTrue(t, pg, vector, "sns")
+	requireQueryTrue(t, pg, vector, "sqs")
+	requireQueryTrue(t, pg, vector, "lol")
+	requireQueryTrue(t, pg, vector, "sos")
+	requireQueryTrue(t, pg, vector, "nope")
+
+	requireQueryFalse(t, pg, vector, "github")
+	requireQueryFalse(t, pg, vector, "spro")
+	requireQueryFalse(t, pg, vector, "snste")
+}
```

Comment:
A couple more negative cases that cross the capitalization boundary would be nice - _zebre_ / _sea_  / _tests_

Reply:
`sea` actually would work because of the size 3 edgram filter we use. Interestingly I tried `seas` and `tests` to test boundaries but they work because of the lemmization of sea  and it being an english word that has the plural form 'seas' and same with `test` being an english word

I did add some more tests though in 4d0a33570d8b524096e3c75208f04a81aee94331 thank you